e2e/framework/README.md

```diff
@@ -0,0 +1,23 @@
+# rook-test-framework
+ 
+Requirements:
+-   Docker 1.12.6 
+-   Ubuntu Host
+ 
+Instructions 
+On your Ubuntu box do the following;
+-   Be sure to allocate atleast 4 processors to your vm
+-   git clone https://github.com/rook/rook
+-   Navigate to the rook directory of the clone sources;
+-         e2e/scripts/smoke_test.sh
+ 
+ 
+ 
+At this point a lot of stuff should be happening on your console output,
+-         create rook-infra container
```

Comment:
rather than show the console output, can you just mention in the paragraph what is happening? i don't think we need the raw output here